### ISSUE TITLE
SessionAuthentication not working anymore for posts, after security updated

### ISSUE THREAD

**Issue by JustinSGray** [django-tastypie/django-tastypie#825]
The SessionAuthentication does not work anymore for posts, after the changes were made to the authorization system.

This is related to the changes made for #332. It seems to show up in commit d850758b088761a00f60a474f5b2d683dc1ec3c0

get still seems to work, but post does not.

**Comment by hmilkovi** [comment]
Yep it doesn't work tryed with ajax 20 ways. Token in header, token i request body, token i cookie with logged in user

**Comment by hmilkovi** [comment]
it's working only needed to put in code:         authorization = Authorization()
